Library/.rubocop.yml

```diff
@@ -28,7 +28,7 @@ FormulaAuditStrict/Options:
   Enabled: true
 
 NewFormulaAudit/Options:
-  Enabled: false
+  Enabled: true 
```

Comment:
I think `FormulaAuditStrict` are fine to leave enabled by default and we just have a separate configuration file for `NewFormulaAudit`. That could be useful for enforcing things like e.g. 80 character wrapping, too.